can-io-firmware/set_servo.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""
+Sends a PWM via UAVCAN
+"""
+
+import argparse
+import uavcan
+import os
+
+DSDL_DIR = os.path.join(os.path.dirname(__file__), '../uavcan_data_types/cvra')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "port",
+        help="SocketCAN interface (e.g. can0) or SLCAN serial port (e.g. /dev/ttyACM0)"
+    )
+
+    parser.add_argument("id", help="ID of the board to target", type=int)
+
+    parser.add_argument("servo", help="Servo output to be controlled", type=int)
+    parser.add_argument("pos", help="Desired duty cycle on the servo output", type=float)
```

Comment:
Maybe provide default values in the script ?

Reply:
Done